src/main/java/com/google/sps/data/Keywords.java

```diff
@@ -0,0 +1,132 @@
+package com.google.sps.data;
+
+import java.io.IOException;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.AnalyzeEntitiesRequest;
+import com.google.cloud.language.v1.AnalyzeEntitiesResponse;
+import com.google.cloud.language.v1.EncodingType;
+import com.google.cloud.vision.v1.EntityAnnotation;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.TreeSet;
+
+/**
+ * Provide methods for the storage, extraction, and retrieval of keywords.
+ */
+public final class Keywords {
+
+  private static final int MAX_NUM_KEYWORDS = 10;
+
+  /**
+   * @return a list of the 10 most salient keywords
+   */
+  public static List<String> getKeywords(String key) throws EntityNotFoundException {
```

Comment:
Is it still throwing this exception?

Reply:
Nope, I seem to have neglected to remove this.